bot/core.py

```diff
@@ -1,207 +1,207 @@
+
+    @commands.command(hidden=True)
+    @commands.check(trusted)
+    async def trust(self, ctx, user: discord.Member):
+        """
+        Adds or removes a user from trusted list
+        """
+        if str(user.id) in fs_data.document("trusted").get().to_dict()["users"]:
+            embed = discord.Embed(title=f"{user.nick if user.nick else user.name} is no longer trusted")
+            fs_data.document("trusted").set(
+                {"users": [str(trustee) for trustee in fs_data.document("trusted").get().to_dict()["users"] if trustee != str(user.id)]})
+        else:
+            embed = discord.Embed(title=f"{user.nick if user.nick else user.name} is now trusted")
+            data = fs_data.document("trusted").get().to_dict()["users"]
+            data.append(str(user.id))
+            fs_data.document("trusted").set({"users": data})
+        embed.set_author(name=ctx.author.nick if ctx.author.nick else ctx.author.name, icon_url=ctx.author.avatar_url)
+        await ctx.message.delete()
+        await ctx.send(embed=embed)
+
+    @commands.command()
+    async def invite(self, ctx):
+        """
+        Gives invite for you to invite bot to your own server
+        """
+        await ctx.message.delete()
+        embed = discord.Embed(title="Add me to your server by clicking here", url="https://discord.com/api/oauth2/authorize?client_id=693313847028744212&permissions=0&scope=bot", colour=BLUE)
+        embed.set_author(name=ctx.author.nick if ctx.author.nick else ctx.author.name, icon_url=ctx.author.avatar_url)
+        await ctx.send(embed=embed)
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        """
+        Sets the status of the bot
+        """
+        activity = discord.Activity(type=discord.ActivityType.watching, name="the rise of Sudan and the fall of noobia")
+        await self.bot.change_presence(status=discord.Status.dnd, activity=activity)
+        await asyncio.sleep(30)
+        await self.bot.change_presence(status=discord.Status.online, activity=activity)
+
+    @commands.command()
+    async def github(self, ctx):
+        """
+        Links to github
+        """
+        embed = discord.Embed(title="Click here to goto my Github", url="https://github.com/TheSuperGamer20578/Sudan-bot", colour=BLUE)
+        embed.set_author(name=ctx.author.nick if ctx.author.nick else ctx.author.name, icon_url=ctx.author.avatar_url)
+        await ctx.message.delete()
+        await ctx.send(embed=embed)
+
+
+def setup(setup_bot):
+    """
+    Initiate cog if loaded as extension
+    """
+    bot.add_cog(core(setup_bot))
+
+
+if __name__ == '__main__':
+    bot = commands.Bot(command_prefix=("&", "/", ".", "sb!", "s!"))  # , commands.when_mentioned))
+    bot.add_cog(core(bot))
+    for cog in config["general"]["autoload cogs"].split("\n"):
+        if cog != "":
+            bot.load_extension(cog)
+    bot.run(config["api"]["discord"])
```

Comment:
Im just going to stop reveiw here and assume its more of the same i dont have time for this